src/Storage.js

```diff
@@ -43,12 +43,16 @@ Storage_.CACHE_EXPIRATION_TIME_SECONDS = 21600; // 6 hours.
 /**
  * Gets a stored value.
  * @param {string} key The key.
+ * @param {boolean} optSkipMemory Whether to bypass the local memory cache when
+ *     fetching the value (the default is false).
  * @return {*} The stored value.
  */
-Storage_.prototype.getValue = function(key) {
-  // Check memory.
-  if (this.memory_[key]) {
-    return this.memory_[key];
+Storage_.prototype.getValue = function(key, optSkipMemory) {
```

Comment:
- "Check memory." doesn't provide more useful information than reading the code.
- Memory is part of every computer instruction, every variable, etc. It'd be better if we use a term like cache to make the distinction clearer (if it doesn't require refactoring code).

```js
Storage_.prototype.getValue = function(key, skipCache) {
  if (!skipCache && this.memory_[key]) {
    return this.memory_[key];
  }
```

Reply:
I'm confused about the rationale for `skipCache`, it's true only inside a slow block (a Lock, must probably followed by an http request) and it only avoid a lookup in an hash. Am I missing something obvious?

(And this PR will breaks #146)

Reply:
Their are corresponding comments for "Check cache" and "Check properties" below. Keeping the comment for parity.

In this code, "cache" refers to a `CacheService` cache, so `skipCache` would by confusing at best. It's more of an in-memory cache (vs being an outside system), but I'd prefer to avoid the word cache to avoid confusion.